desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -163,19 +166,72 @@ def test_check_access_adls(self):
             'context': {}, 
             'operation': {
               'resource': {
-                'storageaccount': 'gethuestorageaccount', 
-                'container': 'demo-gethue-container', 
-                'relativepath': 'demo-dir1/customer.csv'
-              }, 
-              'resourceOwner': 'gethuestorageaccount', 
-              'action': 'read', 
-              'accessTypes': ['read']
+                'storageaccount': 'gethuestorage', 
+                'container': 'data', 
+                'relativepath': '/user/csso_hueuser'
+              },
+              'action': 'get-status', 
+              'accessTypes': ['get-status']
             }
           },
           verify=False
         )
         assert_equal(resp['token'], "nulltenantIdnullnullbnullALLOWEDnullnull1.05nSlN7t/QiPJ1OFlCruTEPLibFbAhEYYj5wbJuaeQqs=")
 
+
+  def test_handle_adls_req_mapping(self):
+    with patch('desktop.lib.raz.raz_client.requests.post') as requests_post:
+      with patch('desktop.lib.raz.raz_client.uuid.uuid4') as uuid:
+        uuid.return_value = 'mock_request_id'
+        requests_post.return_value = Mock(
+          json=Mock(return_value=
+          {
+            'operResult': {
+              'result': 'ALLOWED',
+              'additionalInfo': {
+                "ADLS_DSAS": "nulltenantIdnullnullbnullALLOWEDnullnull1.05nSlN7t/QiPJ1OFlCruTEPLibFbAhEYYj5wbJuaeQqs="
+                }
+              }
+            }
+          )
+        )
+
+        client = RazClient(self.raz_url, self.raz_token, username=self.username, service="adls", service_name="cm_adls", cluster_name="cl1")
+
+        # List directory
+        adls_url = 'https://gethuestorage.dfs.core.windows.net/data?directory=user%2Fcsso_hueuser&resource=filesystem&recursive=false'
+        resp = client.check_access(method='GET', url=adls_url)
+
+        requests_post.assert_called_with(
+          "https://raz.gethue.com:8080/api/authz/adls/access?delegation=mock_RAZ_token",
+          headers={"Content-Type": "application/json"},
+          json={
+            'requestId': 'mock_request_id', 
+            'serviceType': 'adls', 
+            'serviceName': 'cm_adls', 
+            'user': 'gethue', 
+            'userGroups': [], 
+            'clientIpAddress': '', 
+            'clientType': 'adls', 
+            'clusterName': 'cl1', 
+            'clusterType': '', 
+            'sessionId': '', 
+            'accessTime': '', 
+            'context': {}, 
+            'operation': {
+              'resource': {
+                'storageaccount': 'gethuestorage', 
+                'container': 'data', 
+                'relativepath': '/user/csso_hueuser'
+              },
+              'action': 'list', 
+              'accessTypes': ['list']
```

Comment:
We might want to only check for those only, as the test methods start to be pretty bloated